core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.spec.ts

```diff
@@ -1430,6 +1562,290 @@ describe('EditEmaEditorComponent', () => {
             });
         });
 
+        describe('move contentlet', () => {
+            it('should post to iframe to get bound on move contentlet', () => {
+                spectator.detectChanges();
+
+                const iframe = spectator.debugElement.query(By.css('[data-testId="iframe"]'));
+
+                const postMessageSpy = jest.spyOn(
+                    iframe.nativeElement.contentWindow,
+                    'postMessage'
+                );
+
+                window.dispatchEvent(
+                    new MessageEvent('message', {
+                        origin: HOST,
+                        data: {
+                            action: CUSTOMER_ACTIONS.SET_CONTENTLET,
+                            payload: {
+                                x: 100,
+                                y: 100,
+                                width: 500,
+                                height: 500,
+                                payload: PAYLOAD_MOCK
+                            }
+                        }
+                    })
+                );
+
+                spectator.detectChanges();
+
+                const emaTools = spectator.debugElement.query(
+                    By.css('[data-testId="contentlet-tools"]')
+                );
+
+                spectator.triggerEventHandler(emaTools, 'move', {
+                    ...PAYLOAD_MOCK
+                });
+
+                spectator.detectComponentChanges();
+
+                expect(postMessageSpy).toHaveBeenCalledWith('ema-request-bounds', '*');
```

Comment:
Don't you need to check that the dropzone is visible?

Reply:
Yes you are right!